ds3-sdk-integration/src/test/java/com/spectralogic/ds3client/integration/GetJobManagement_Test.java

```diff
@@ -0,0 +1,150 @@
+import java.nio.channels.WritableByteChannel;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.security.SignatureException;
+import java.util.UUID;
+
+import static com.spectralogic.ds3client.integration.Util.RESOURCE_BASE_NAME;
+import static com.spectralogic.ds3client.integration.Util.deleteAllContents;
+import static org.hamcrest.Matchers.*;
+import static org.junit.Assert.*;
+
+public class GetJobManagement_Test {
+
+    private static final Ds3Client client = Util.fromEnv();
+    private static final Ds3ClientHelpers HELPERS = Ds3ClientHelpers.wrap(client);
+    private static final String BUCKET_NAME = "Get_Job_Management_Test";
+    private static final String TEST_ENV_NAME = "GetJobManagement_Test";
+    private static TempStorageIds envStorageIds;
+
+    @BeforeClass
+    public static void startup() throws Exception {
+        final UUID dataPolicyId = TempStorageUtil.setupDataPolicy(TEST_ENV_NAME, true, ChecksumType.Type.MD5, client);
+        envStorageIds = TempStorageUtil.setup(TEST_ENV_NAME, dataPolicyId, client);
+        HELPERS.ensureBucketExists(BUCKET_NAME);
+        putBeowulf();
+    }
+
+    @AfterClass
+    public static void teardown() throws IOException, SignatureException {
+        deleteAllContents(client, BUCKET_NAME);
+        TempStorageUtil.teardown(TEST_ENV_NAME, envStorageIds, client);
+        client.close();
+    }
+
+    private static void waitForObjectToBeInCache(final UUID jobId) throws Exception {
+        long cachedSize = 0;
+        int cycles = 0;
+        while (cachedSize == 0) {
+            Thread.sleep(500);
+            final MasterObjectList mol = client.getJobSpectraS3(new GetJobSpectraS3Request(jobId)).getMasterObjectListResult();
+            cachedSize = mol.getCachedSizeInBytes();
+            cycles++;
+            if (cycles > 20) {
+                throw new Exception("Failed to upload object to cache in 10 seconds");
+            }
+        }
+    }
+
+    private static void putBeowulf() throws Exception {
+        final String book1 = "beowulf.txt";
+        final Path objPath1 = ResourceUtils.loadFileResource(RESOURCE_BASE_NAME + book1);
+        final Ds3Object obj = new Ds3Object(book1, Files.size(objPath1));
+        final Ds3ClientHelpers.Job job = HELPERS.startWriteJob(BUCKET_NAME, Lists
+                .newArrayList(obj));
+        final UUID jobId = job.getJobId();
+        final SeekableByteChannel book1Channel = new ResourceObjectPutter(RESOURCE_BASE_NAME).buildChannel(book1);
+        client.putObject(new PutObjectRequest(BUCKET_NAME, book1, book1Channel, jobId, 0, Files.size(objPath1)));
+        waitForObjectToBeInCache(jobId);
+    }
+
+    @SuppressWarnings("deprecation")
+    @Test
+    public void nakedS3Get() throws IOException, SignatureException, XmlProcessingException,
+            URISyntaxException, InterruptedException {
+
+        final WritableByteChannel writeChannel = Channels.newChannel(new OutputStream() {
```

Comment:
We have a NullChannel implementation that does what you want here.  Use that instead.